src/containers/BooksForm.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
+
+const BooksForm = ({ createBook }) => {
+  const categories = ['Action', 'Biography', 'History',
+    'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+  const [values, setValues] = useState({
+    id: '',
+    title: '',
+    category: 'Action',
+  });
+
+  const handleChange = (e) => {
+    setValues((values) => ({
+      ...values,
+      [e.target.name]: e.target.value,
+    }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const { title, category } = values;
+
+    createBook({
+      id: Math.floor(Math.random() * 1111).toString(),
+      title,
+      category,
+    });
+
+    setValues({
+      id: '',
+      title: '',
+      cartegory: '',
```

Comment:
- [ ] [REQUIRED] You should reset the `category` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)). It means that the value of `category` should be equal to `Action`.